### PR TITLE
Update xmlDom dependency to fix security issues

### PR DESCRIPTION
**Pull request by pang0018**
This PR changes the xmlDom library dependency to not be embedded and to use the @xmldom/xmldom library. This resolves security issues with plist library

https://github.com/TooTallNate/plist.js/issues/136

### CODE REVIEW

File: dist/plist-parse.js [hunk 1]
```diff
@@ -229,7 +229,282 @@ function parsePlistXML (node) {
 }
 
 }).call(this)}).call(this,require("buffer").Buffer)
-},{"./xmldom/dom-parser":2,"buffer":7}],2:[function(require,module,exports){
+},{"@xmldom/xmldom":6,"buffer":9}],2:[function(require,module,exports){
```
Comment (pang0018) [inline dist/plist-parse.js:232]: Not sure why all this changed, I just ran 'make' and this updated. Can revert if necessary.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "plist",
   "description": "Apple's property list parser/builder for Node.js and browsers",
-  "version": "3.0.6",
+  "version": "3.0.7",
```
Comment (mreinstein) [inline package.json:4]: don't bump the version number in this PR please, that'll be done separately.

**Review by mreinstein — CHANGES_REQUESTED** [review]

**Comment by mreinstein** [comment]
Thanks for the PR!

File: package.json (hunk 2 above)
Comment (pang0018) [reply package.json]: Reverted. Ready for another look!

**Review by mreinstein — APPROVED** [review]

**Comment by pang0018** [comment]
@mreinstein ty for merging this change so quickly. When will a new release with this change come out?

**Comment by mreinstein** [comment]
Done. Published as 3.1.0.

Thanks again for the PR! ❤️

**Pull request merged**